Dependencies/xr/Include/XR.h

```diff
@@ -185,6 +200,8 @@ namespace xr
                     const Identifier ID{ NEXT_ID++ };
                     bool TrackedThisFrame{};
                     bool JointsTrackedThisFrame{};
+                    bool GamepadEnabledThisFrame{};
```

Comment:
Would it make more sense to call this GamepadTrackedThisFrame, to match the others above it?